src/wtforms/validators.py

```diff
@@ -594,10 +594,20 @@ def __init__(self, values, message=None, values_formatter=None):
         self.values_formatter = values_formatter
 
     def __call__(self, form, field):
+        if field.type == "SelectMultipleField":
+            if any(e in self.values for e in field.data):
+                message = self.message
+                if message is None:
+                    message = field.gettext('Invalid value, cannot be any of: %(values)s.')
+
+                raise ValidationError(
+                    message % dict(values=self.values_formatter(self.values))
+                )
+
         if field.data in self.values:
```

Comment:
If the field is `SelectMultipleField`, this behavior still triggers if the new if block doesn't raise. That would sort of address my point about validating lists vs items, but it doesn't seem like the right design because there's still no way to control *which* type of validation you intended.